src/kubernetes.cc

```diff
@@ -705,14 +705,22 @@ json::value KubernetesReader::QueryMaster(const std::string& path) const
 }
 
 namespace {
-struct Watcher {
+static std::shared_ptr<boost::asio::io_service>& Reset(
```

Comment:
My fault — I initially put this inside the `Watcher` class, and forgot to remove `static` after moving it out.

Reply:
Done.